scripts/jenkins-build.sh

```diff
@@ -30,8 +30,14 @@ echo CCPi build
 echo called with arguments: $@
 echo CCPI_BUILD_ARGS: $CCPI_BUILD_ARGS
 
-#print GPU driver info
-nvidia-smi
+# If NO_GUI = true then don't print GPU driver info
+if [[ ! -n ${NO_GUI} ]] || [ ${NO_GUI} = false ]; then
```

Comment:
Why do you need the additional OR  testing if `NO_GUI` is false?

Reply:
Just thought that we should be printing the info if NO_GUI = false, otherwise it wouldn't be because it's just checking whether NO_GUI is set to anything. Fine to change this though if you like.